package.json

```diff
@@ -34,8 +32,16 @@
     "node-sass": "4.13.1",
     "pepjs": "0.5.2",
     "prettier": "1.19.1",
+    "react": "16.13.0",
+    "react-dom": "16.13.0",
```

Comment:
Why did you move react to devDependencies?

Reply:
As a component, react and react-dom are required as peerDeps (use instances of host).
If this repo has those, it makes confusing at build step.